lib/obsolete.gi

```diff
@@ -879,18 +879,6 @@ end );
 
 #############################################################################
 ##
-#F  USER_HOME_EXPAND
```

Comment:
`USER_HOME_EXPAND` is not being removed, just switched to using `DeclareObsoleteSynonym`.